modules/dashboard/php/taskwidget.class.inc

```diff
@@ -89,7 +109,7 @@ class TaskWidget implements \LORIS\GUI\Widget
      */
     public function siteLabel() : string
     {
-        return "";
+        return $this->sitelabel;
```

Comment:
This is still going to change the behaviour  on any taskwidgets that don't call setSiteLabel since it changed from the empty string to null and I'm still not crazy about the setter.

All the other parameters are injected via the constructor, and you call setSiteLabel immediately after the constructor in the only place it's used. Why don't you pass a site label in the constructor with a default value of ""? I think that would take care of both problems without changing any behaviour. of widgets in any other modules that don't call setSiteLabel.

Reply:
Changed in last commit.